cylc/flow/scripts/workflow_state.py

```diff
@@ -18,252 +18,404 @@
 
 r"""cylc workflow-state [OPTIONS] ARGS
 
-Retrieve task states from the workflow database.
+Deprecated support for existing scripts:
+    cylc workflow-state --task=NAME --point=CYCLE --status=STATUS
+       --output=MESSAGE --message=MESSAGE WORKFLOW
```

Comment:
This shouldn't be mentioned until later (currently breaks the `cylc help all` description)

Reply:
Oops yes, fixed.